cmake/Codegen.cmake

```diff
@@ -235,11 +235,11 @@ function(append_filelist name outputvar)
   set(_rootdir "${${CMAKE_PROJECT_NAME}_SOURCE_DIR}/")
   # configure_file adds its input to the list of CMAKE_RERUN dependencies
   configure_file(
-      ${CMAKE_SOURCE_DIR}/tools/build_variables.bzl
-      ${CMAKE_BINARY_DIR}/caffe2/build_variables.bzl)
+	  ${PROJECT_SOURCE_DIR}/tools/build_variables.bzl
+	  ${PROJECT_BINARY_DIR}/caffe2/build_variables.bzl)
```

Comment:
lint is failing you because of tabs here, please fix?

Reply:
will push up an updated patch.